addons/edi_stock/models/edi_move_cancellation_report_document.py

```diff
@@ -20,6 +20,13 @@ class StockMove(models.Model):
         index=True,
     )
 
+    cancelled_by = fields.Many2one("res.users", string="Cancelled By", readonly=True)
+
+    def _action_cancel(self):
+        User = self.env["res.users"]
+        self.cancelled_by = User.browse(self.env.real_uid)
```

Comment:
nit: I think there should be a space between these two lines so like: 
```suggestion
         User = self.env["res.users"]

        self.cancelled_by = User.browse(self.env.real_uid)
```

Reply:
second nit: might be able to change to just
```python
self.cancelled_by = self.env.real_uid
```

Reply:
I believe this returned an int. I could be mistaken though

Reply:
Yeah it is an int.

So in env you have `user` and `uid`, the UDES 11 version of Odoo only added in the ID of the real user.

On that note though... `real_uid` isn't a thing in vanilla Odoo... might need some thinking about.

Reply:
This will need reviewing.

Reply:
Thanks for sorting @jrbirch99 - just took a while to think of the nicest/least worse solution! Looks good just one nit.